### DISCUSSION TITLE
Virtual Team Meeting - 2021-10-20

### DISCUSSION THREAD

**Discussion by jwildfire** [SafetyGraphics/safetyGraphics#644] [Meetings]
There were lots of conflicts with today's meeting, but did want to provide some quick updates so we thought we'd have a "Virtual Meeting" this month, so feel free to reply with any updates or questions on this thread. 

Here are a few notes from my end: 

- We've got 30+ participants registered for the safetyGraphics workshop at [R/Pharma](https://rinpharma.com/) on Oct 29. We created a [new repo](https://github.com/SafetyGraphics/RPharma2021-Workshop) to save the materials for the conference. Many thanks to @xni7, @rescalan, @MayaGans, @elimillera, @samussiah and @bzkrouse for all their help! 
- We're also running a virtual booth at the virtual expo for the R/Pharma virtual conference. So stop by and chat if you're attending. 
- Our workshop prep has given us a few ideas for the next release of safetyGraphics - v2.1 will focus on streamlining the process for adding additional data domains to the app, and I'm hoping we can release it before the end of the year. The project board with details and links to issues is [here](https://github.com/SafetyGraphics/safetyGraphics/projects/15).
- On a related note, we've started work on several new charts (some in new data domains!) in the safetyCharts package. @xni7  - maybe you can reply with details on the ecg/QT work? 
- I gave a presentation and workshop on safetyGraphics at a [PSI Single Day Event](https://psiweb.org/events/event-item/2021/09/17/default-calendar/psi-scientific-meeting-generating-insights-through-modern-applications-of-data-visualisation) last month. @spduke also gave a nice talk on graphics best practices! We gots of good feedback on our work (and possibly a few new group members!). 
- We've made a few updates in our vignettes. Brian (@waterhob) caught a bunch of typos (thanks again, Brian!) and we added [a new question in the FAQ](https://github.com/SafetyGraphics/safetyGraphics/wiki/TechnicalFAQ#q-is-the-safetygraphics-app-ready-for-production-use) discussing 'Is the safetyGraphics app ready for "production" use?'
- We also found 2 bugs this month. Click through to the links for full discussions. We'll get both of these resolved ASAP (certainly by the end of the year)
   - https://github.com/SafetyGraphics/safetyGraphics/issues/642 
   - https://github.com/SafetyGraphics/hep-explorer/issues/330

Hope everyone is having a good month!

**Comment by xni7** [comment]
nice summary @jwildfire ! Some quick updates on QT/ECG

- **ECG ADaM data:**  we found some [nice sample ADaM data](https://physionet.org/content/ecgcipa/1.0.0/) from [an FDA sponsored thorough QT study](https://clinicaltrials.gov/ct2/show/NCT03070470). 
- As part of the workshop prep, we prototyped a [QT app](https://xiao-ni.shinyapps.io/safetyGraphics_QT/) using this ADEG data set and the V2 framework. Here is [code snippet](https://gist.github.com/xni7/8bfa91f4d7f809d0d3d7c1954c360957) to launch the app from the console. This is a quick & dirty approach by repurposing lab charts to QT data given the similarity of data structure. The QT charts include the typical lab charts such as histogram, boxplots, shift plots, outlier explorer, paneled outlier plot, plus a [new QT Outlier Explorer](https://github.com/SafetyGraphics/safetyCharts/blob/dev/R/QT_Outlier_Explorer.R), inspired by @anzianorj 's CTSpedia QT plot [link broken](https://www.ctspedia.org/wiki/pub/CTSpedia/StatGraphTopic017). 
- **Adding QT charts to `safetyCharts`**: we formalized the prototype into `safetyCharts` by adding a new [ECG domain in the meta data](https://github.com/SafetyGraphics/safetyCharts/blob/dev/data-raw/meta_ecg.csv). More details found in [this pull request](https://github.com/SafetyGraphics/safetyCharts/pull/90)

@jwildfire @JimBuchanan @mli1 @anzianorj feel free to add more details.
